internal/notify/mouse.go

```diff
@@ -2,6 +2,7 @@ package notify
 
 import "fyne.io/fyne"
 
+// MouseNotify is an interface that lets separate packages like wm tell the desktop that the cursor has moved to or from the desktop canvas.
```

Comment:
Not sure about this - it has no connection to the window manger, it's just an interface?
How about:
"MouseNotify is use to mark an object that is interested in when the mouse enters or exists the desktop"